core/src/test/java/com/expediagroup/streamplatform/streamregistry/core/events/NotificationEventEmitterConsumerServiceTest.java

```diff
@@ -217,17 +206,19 @@ public void givenAConsumerForCreate_handleAMulticasterException() {
   }
 
   public <T> NotificationEvent<T> getDummyNotificationEvent(String source, EventType type, T entity) {
-    return NotificationEvent.<T> builder()
+    return NotificationEvent.<T>builder()
         .source(source)
         .eventType(type)
         .entity(entity)
         .build();
   }
 
+  private final ObjectMapper mapper = new ObjectMapper();
+
   public Specification getDummySpecification() {
     Specification spec = new Specification();
-    spec.setConfiguration(deserialise("{}"));
+    spec.setConfiguration(mapper.createObjectNode());
```

Comment:
If these two approaches ever diverge, there could be problems created.
e.g. what if at some point we either don't use this mapper, or we customise it's behaviour.
Not a big problem here I guess, since it is a test.